AudioKit/Core Classes/AKManager.h

```diff
@@ -29,6 +29,9 @@ NS_ASSUME_NONNULL_BEGIN
 /// Determines whether or not to log
 @property BOOL isLogging;
 
+/// Place to put lines of testing logs
+@property NSMutableArray *testLog;
+
```

Comment:
I'm a bit concerned about keep log lines in memory - there's a real risk of a runaway log just crashing things if it goes out of control.

Also if this is only for unit tests, maybe we can use `#ifdef TRAVIS_CI` to only include this piece then.